packages/evidence/cli.js

```diff
@@ -368,8 +368,8 @@ prog
 		const flatArgs = flattenArguments(args);
 
 		logQueryEvent('preview-server-start', undefined, undefined, undefined, true);
-		// Run svelte kit dev in the hidden directory
-		const child = spawn('npx vite preview --outDir build --port 3000', flatArgs, {
+		// We will likely need to modify this for SPA mode previews
+		const child = spawn('npx serve build', flatArgs, {
```

Comment:
`serve -s` for SPA mode

Reply:
I would really prefer to just use their [API](https://www.npmjs.com/package/serve#API) instead of forking a child process, but if that's too much on this PR then this seems okay

Reply:
@mcrascal 
`serve -s` is correct for SPA mode, but it breaks non spa mode on navigation, so we will need to read the config option into this file to support it